public/locales/de/faq.json

```diff
@@ -0,0 +1,47 @@
+{
+  "title": "FAQ - Hier findet Ihr Antworten auf Eure Fragen.",
+  "hospital": {
+    "title": "Krankenhaus / Soziale Einrichtung",
+    "ask1": "Ich bin Arzt, Krankenhaus, Pflege-, soziale oder öffentliche Einrichtung. Wie kann ich Material anfragen?",
+    "ans1": {
+      "part1": "Dazu müsst Ihr euch einloggen und bei dem gewünschten Teil rechts auf das",
```

Comment:
These values can be placed in 1 key. in other languages this might be 4 parts or 8 ;-) So no need for separating them.

Reply:
sure, i planned to do it like this: [https://react.i18next.com/latest/trans-component](https://react.i18next.com/latest/trans-component)